_announcements/week-07.md

```diff
@@ -0,0 +1,71 @@
+date: 2023-02-24
+---
+
+# 🎰 Week 7: Lists and Dictionaries
+
+Happy Weekend! 
+
+We hope you all had a great week! With just 1/3rd of the course left, we hope you are making sure that you are taking good care of yourselves before we get into the final stretch of the quarter.
+
+ - Week 7 was about working with lists and dictionaries, and the different methods that we can use with them.
+    - Helpful list methods included, for example, `append()`, `insert()`, `pop()`, `extend()`, `sort()` (in place!) 
+    - Dictionaries used `get()`, `update()`, `pop()`, `clear()`, and much more. 
+    -  We encourage you to practice writing code from scratch using different methods for lists and dictionaries.
+ - During labs, we went over some important concepts on lists and dictionaries and how to use different methods with them, like list slicing, list iteration etc. We also reviewed nested lists.
+
+## ⏰ Week 8 - Due dates and Deadlines ⏰
+
+The deadlines **next week** are:
+* ⏰ **Tuesday, 02/28 11:59PM** Due: **LA07**
+* ⏰ **Wednesday, 03/01 11:59PM** Due: **LA08 CHECKPOINT labs** - Week 8 zyLabs with the word “Checkpoint” in their title
+* ⏰ **Thursday, 03/02 11:59PM** in-class Quiz 6 (starts on Gauchospace, file submission to Gradescope). Quiz covers Week 1-7 content.
+* ⏰ **Saturday, 03/04 11:59PM** Due: **Weekly reflection** posted on Gauchospace + **PA08** + **CA08**
+
+Quiz 5 has been graded and released - you should have received an email from Gradescope titled "_Your Quiz 5 (week7) for CSW 8, Spring 2022 has been graded_".
+* ⏰ **Wednesday 10PM** Regrade requests are Due on **Gradescope**
+    * Note regarding regrade-requests: After the quiz grades are released, students can submit a regrade request on Gradescope as described here: [{{site.url}}/{{site.baseurl}}/quiz]({{site.url}}/{{site.baseurl}}/quiz).
+    * For Gradescope quizzes always use the regrade request button (**not email or the forum**).
+* We also added [Common autograder error messages on Gradescope]({{site.url}}/{{site.baseurl}}/ref/debug/#common-autograder-error-messages-on-gradescope) to help you makes sense of some of the errors that you might be seeing there.
+
+
+## ➡️    Staying on Track in Week 8
+
+🔮 [Looking ahead to Week 8]({{site.url}}/{{site.baseurl}}/calendar#week-8), your tasks for the upcoming days are:
+* Remember that unlike lists and dictionaries **strings are immutable**! 💎
+    - Note that string methods **return a new string object** instead of modifying the string you asked it to operate on.
+    - The "📎 Quick access" notes within the zyBooks sections point you to the resources you will definitely need to reference, so make a note of them.
+* Write down anything that's unclear and if you have any questions to ask during class
+* Practice writing code from scratch using your IDE.
+
+## Reminder for TA Evaluation form
+
+* If you haven't already, here is the link to TA evaluation form: https://forms.gle/Wa4mXoAnh8n647499. We would appreciate your feedback!
```

Comment:
In the draft for the Piazza post, we had some info on why it is important/valuable to us. 

Also, links need to use markdown, otherwise, they won't be clickable.